internal/ascprails/rails.go

```
@@ -0,0 +1,466 @@
+	return s.store.Enqueue(ctx, input)
+}
+
+// DispatchOne advances exactly one durable dispatch job. SENDING is persisted
+// before network I/O; an expired lease may replay only the exact same call ID
+// and payment proof, which the escrow-call seller contract must deduplicate.
+func (s *Service) DispatchOne(ctx context.Context) (Job, error) {
+	lease, err := s.store.ClaimDispatch(ctx, s.config.WorkerID, s.config.LeaseDuration)
+	if err != nil {
+		return Job{}, err
+	}
+	defer s.store.ReleaseLease(context.WithoutCancel(ctx), lease)
```

golangci-lint (2.12.2)

[error] Error return value of `s.store.ReleaseLease` is not checked

(errcheck)